client/me/help/help-contact/index.jsx

```diff
@@ -64,6 +65,10 @@ const HelpContact = React.createClass( {
 			this.props.connectHappychat();
 		}
 
+		if ( config.isEnabled( 'help/directly' ) ) {
+			this.props.initializeDirectly();
```

Comment:
@mtias I cant see what that would look like in this case. `initializeDirectly` is an action creator so are you saying we should asyncrequire the actions? AFAIK the actual directly code is "required" [here](https://github.com/Automattic/wp-calypso/blob/b4658c5b57a6d1ed2799a8ab68a0b4012a10a7eb/client/state/data-layer/third-party/directly/index.js#L8) in third party middleware. Maybe we can chat about it in slack a bit and that would help clarify concerns, etc.